GVFS/GVFS.Common/Tracing/JsonTracer.cs

```diff
@@ -83,13 +86,22 @@ public bool HasLogFileEventListener
 
         public void AddEventListener(EventListener listener)
         {
+            if (this.isDisposed)
+            {
+                Console.WriteLine("Adding listener to disposed tracer");

```

Comment:
In the "common" case, the mount process will not have a console...  I think Json tracer might already write to console, but is this what we want?

Reply:
Will remove this. I added it to be consistent with the other 'is disposed' test that already existed in the file. Looks like that was debugging code that shouldn't have been in there in the first place. Will remove this.